x-pack/auditbeat/module/system/login/utmp.go

```diff
@@ -94,6 +94,7 @@ func (r *UtmpFileReader) ReadNew() (<-chan LoginRecord, <-chan error) {
 	errorC := make(chan error)
 
 	go func() {
+		defer logp.Recover("recovering from a crash while collecting login information")
```

Comment:
The message that gets appended already states that it's recovering. So how about: `A panic occurred while collection login information`